src/logic/puzzles.js

```diff
@@ -2204,292 +2204,4368 @@ export const puzzles = [
+      "basic",
+      "portal",
+      "outer",
+    ],
+  },
+  {
+    station: "Zigger Station",
+    room: "6",
+    startingText: "Try zigging and zagging!",
+    winText:
+      "Nice job! Let us know what you thought of the game at www.patreon.com/c/skedwards88. We’re still in beta, so all feedback is appreciated!",
+    robotStartMood: "happy",
+    robotEndMood: "happy",
+    puzzle: [
+      "outer",
+      "portal",
+      "flask",
+      "exit",
+      "flask",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "2",
+      "basic",
+      "4",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "flask",
+      "jet",
+      "1",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "5",
+      "jet",
+      "3",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "basic",
+      "outer",
+      "basic",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "flask",
+      "start",
+      "flask",
+      "portal",
+      "outer",
+    ],
+  },
+  {
+    station: "Zigger Station",
+    room: "7",
+    startingText: "Try zigging and zagging!",
+    winText:
+      "Nice job! Let us know what you thought of the game at www.patreon.com/c/skedwards88. We’re still in beta, so all feedback is appreciated!",
+    robotStartMood: "happy",
+    robotEndMood: "happy",
+    puzzle: [
+      "outer",
+      "portal",
+      "basic",
+      "exit",
+      "basic",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "door",
+      "outer",
+      "key",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "basic",
+      "flask",
+      "basic",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "key",
+      "outer",
+      "door",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "basic",
+      "flask",
+      "basic",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "flask",
+      "outer",
+      "key",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "door",
+      "flask",
+      "basic",
+      "portal",
+      "outer",
+      "outer",
+      "outer",
+      "basic",
+      "outer",
+      "flask",
+      "outer",
+      "outer",
+      "outer",
+      "portal",
+      "basic",
+      "start",
+      "basic",
+      "portal",
+      "outer",
+    ],
+  },
+  {
+    station: "Cube station",
+    room: "1",
+    startingText:
+      "By beating the story, you've unlocked spectial stations! This one's actually just a square, but you'll have to think outside the box to get through.",
```

Comment:
do you mean to omit "By beating the story, you've unlocked spectial stations!" If not, need to fix "spectial" spelling.